web/src/js/ducks/ui/flow.js

```diff
@@ -87,10 +98,27 @@ export function updateEdit(update) {
     return { type: UPDATE_EDIT, update }
 }
 
-export function stopEdit(flow) {
+export function uploadContent(flow, content, type){
```

Comment:
This looks like a second action creator that does exactly the same thing as `updateContent` but emits an additional event. I don't think that separation is useful.